hack/local-up-cluster.sh

```diff
@@ -704,7 +706,11 @@ function start_kubelet {
     mkdir -p "/var/lib/kubelet" &>/dev/null || sudo mkdir -p "/var/lib/kubelet"
     # Enable dns
     if [[ "${ENABLE_CLUSTER_DNS}" = true ]]; then
-      dns_args="--cluster-dns=${DNS_SERVER_IP} --cluster-domain=${DNS_DOMAIN}"
+      if [[ "${ENABLE_NODELOCAL_DNS}" = true ]]; then
```

Comment:
`"${ENABLE_NODELOCAL_DNS:-}"` and quote true

Reply:
done